recipe/build.sh

```diff
@@ -19,14 +19,12 @@ if [ $(uname) == Linux ]; then
     patch -p0 < "${RECIPE_DIR}"/0005-qtwebkit-fix-TEXTREL-on-x86-changeset_172759.patch
     rm qtwebkit.tar
 
-    MAKE_JOBS=$CPU_COUNT
-
     ./configure -prefix $PREFIX \
                 -libdir $PREFIX/lib \
                 -bindir $PREFIX/bin \
                 -headerdir $PREFIX/include/qt \
-                -archdatadir $PREFIX \
-                -datadir $PREFIX \
```

Comment:
Why are you changing this? I think it will break every other lib that depends on Qt.

Reply:
I though it was cleaner to install plugins & stuff in a subdir, are you sure this will break something ? libs are still installed in libdir

Reply:
I reverted in doubt